bletl/growth.py

```diff
@@ -37,10 +45,10 @@ def __init__(
             backscatter vector
         switchpoints : dict
             maps switchpoint times to labels
-        pmodel : pymc3.Model
-            the PyMC3 model underlying the analysis
+        pmodel : pymc.Model
```

Comment:
here you use `pymc `where everywhere else you use `pm`.

Reply:
Or is it since you want to make clear in the docstrings that it is PyMC (regardless of version)?

Reply:
Yes, I wanted them to be more human-friendly this way.